### PR TITLE
Migrate from jenkins to GitHub actions

### PR DESCRIPTION
**Pull request by joakimen**
Migrates to GHA. Adds a disable-step in `ci.yml` that must be manually removed when application has been updated and is ready to produce artifacts.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  build-and-deploy:
+    timeout-minutes: 60
+    runs-on: ubuntu-22.04
+    permissions:
+      actions: read
+      contents: read
+      id-token: write
+    steps:
+
+      # - name: disable job
```
Comment (stekern) [inline .github/workflows/ci.yml:33]: Should this instead be a conditional step that only runs if the name of the repository is something else? `if: ${{ github.event.repository.name != 'liflig-cdk-app-reference' }}`

**Review by stekern — COMMENTED** [review]
One suggestion, besides that this looks good 👌

File: .github/workflows/ci.yml (hunk 1 above)
Comment (joakimen) [reply .github/workflows/ci.yml:33]: Yeah probably, makes it less annoying to test in this repo. Fixing

Comment (joakimen) [reply .github/workflows/ci.yml]: https://github.com/capralifecycle/liflig-cdk-app-reference/pull/64/commits/b195e6ab491ad850537e41b3aa820001d00ae791

**Review by stekern — APPROVED** [review]
Gucci 🥇

**Pull request merged**